dimod/core/sampler.py

```diff
@@ -115,6 +122,8 @@ class Sampler(metaclass=SamplerABCMeta):
     Provides all methods :meth:`~.Sampler.sample`, :meth:`~.Sampler.sample_ising`,
     :meth:`~.Sampler.sample_qubo` assuming at least one is implemented.
 
```

Comment:
According to comment https://github.com/dwavesystems/dimod/pull/799#discussion_r577249029, the implementation does not have to call the method but below you say ", which  should be called in each implemented sample method"